samples/minimum_criteria.rst

```diff
@@ -0,0 +1,82 @@
+.. _definition_and_minimum_criteria:
+
+Sample Definition and Minimum Criteria
+######################################
+
+Definition
+==========
+
+A sample is a basic Zephyr application that provides a brief overview of a feature, subsystem, or module. For example, Kernel services, drivers, protocols, etc.
+Samples are limited in scope and should focus only on capturing the nontrivial and introductory aspects of the functionality in question.
+
+Minimum Requirements/ Criteria for Samples:
+===========================================
+
+1. Samples must not use Zephyr's testing framework.
```

Comment:
You are using a mix of 'must not', 'shall not' and should, please use one form.

Reply:
Have revised this as suggested.